core/src/main/java/org/apache/calcite/plan/SubstitutionVisitor.java

```diff
@@ -1070,6 +1071,70 @@ private ScanToCalcUnifyRule() {
     }
   }
 
+  /**
+   * A {@link SubstitutionVisitor.UnifyRule} that matches a {@link MutableCalc}
+   * which has {@link MutableAggregate} as  child.
```

Comment:
What's going on here ? 
This doc here seems not consistent with the implementation.